website/docs/d/project_ip_whitelist.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: project_ip_whitelist"
+sidebar_current: "docs-mongodbatlas-resource-project-ip-whitelist"
```

Comment:
Since it's a data source shouldn't this be sidebar_current: "docs-mongodbatlas-datasource-project-ip-whitelist"?